case-study-1-food-delivery-example-1/functional/test_orders.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+import requests
+import os
+import sys
+
+examples_path = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+sys.path.append(examples_path)
+
+import helper
+helper = helper.Helper("case-study-1-food-delivery-example-1")
+
+# Note that tests should be prefixed with test_functional for filibuster compatibility
+def test_create_update_delete():
+    response = requests.get("{}/orders/create".format(helper.get_service_url('orders')), timeout=helper.get_timeout('orders'))
```

Comment:
You should have a single test for each of these: a test for create, a test for update, and a test for delete.